5_databases/week3/homework3.sql

```diff
@@ -0,0 +1,85 @@
+-- Active: 1706363831540@@0.0.0.0@3306@mealsharing
+
+CREATE TABLE meal (
+    id INT(6) AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(50),
+    description TEXT,
+    location VARCHAR(100),
+    `when` DATETIME,
+    max_reservation INT(6),
+    price DECIMAL,
+    create_date DATE
+);
+
+CREATE TABLE reservation (
+    id INT(6) AUTO_INCREMENT PRIMARY KEY,
+    number_of_guests INT(6),
+    meal_id INT(6),
+    FOREIGN KEY (meal_id) REFERENCES meal(id),
+    created_date DATE,
+    contact_phonenumber VARCHAR(20),
+    contact_name VARCHAR(50),         
+    contact_email VARCHAR(100)        
+);
+
+CREATE TABLE review (
+    id INT(6),
+    title VARCHAR(20),
+    description TEXT,
+    meal_id INT(6),
+    FOREIGN KEY (meal_id) REFERENCES meal(id),
+    stars INT,
+    create_date DATE
+);
```

Comment:
More like a suggestion: 
1) it's better to assign foreign keys at the end. you can have a few of them and it's just more readable 
2) you could also add `NOT NULL` for columns that should be filled, fx. id should always be defined